recipes/level-zero/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,5 @@
+#include "ze_api.h"
+
+int main() {
+    zeInit(0);
```

Comment:
```suggestion
    zeInit(0);
    return 0;
```

Avoid compiler warnings. The main method expects returning an integer.